client/package.json

```diff
@@ -8,7 +8,9 @@
     "@testing-library/user-event": "^12.6.0",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "react-scripts": "4.0.1",
+    "react-native": "^0.63.4",
+    "react-scripts": "^4.0.1",
+    "react-scroll": "^1.8.1",
```

Comment:
Please remove extra libraries that we dont need.

Reply:
okay sir